sections/futures/Trade/TransferIsolatedMarginModal.tsx

```diff
@@ -26,10 +31,15 @@ type Props = {
 	defaultTab: 'deposit' | 'withdraw';
 };
 
+const SocketBridge = dynamic(() => import('../../../components/SocketBridge'), {
+	ssr: false,
+});
+
 const PLACEHOLDER = '$0.00';
 
 const TransferIsolatedMarginModal: React.FC<Props> = ({ onDismiss, defaultTab }) => {
 	const { t } = useTranslation();
+	const { walletAddress } = Connector.useContainer();
```

Comment:
I think we should make it good practice to fetch this from redux state (selectWallet) rather than connector, we'll probably remove it from connector eventually to make sure we only have a single data source. This would help with things like if we wanted to allow users to view the app from a selected wallet etc.

Reply:
Yes.Thanks for pointing it out.